README.md

```diff
@@ -14,11 +14,17 @@ for Pry.
 Installation
 ------------
 
-All you need is to install the gem. The `pry-doc` plugin will be detected and
-used automatically.
+Install the gem using the below command.
 
     gem install pry-doc
 
+To use `pry-doc`, you will need to manually require the gem inside `pry`.
```

Comment:
Should be required from your `pryrc`.

Reply:
In other words, I wouldn't mention this at all. Just the need to require from `pryrc`